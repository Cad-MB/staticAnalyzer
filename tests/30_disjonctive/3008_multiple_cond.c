{
    int x;
    x = 0;
    if (x <= 2) {
      x = 1;
    } else if (x <= 5) {
      x = 4;
    } else {
      x = 7;
    }
    print_all;
  }