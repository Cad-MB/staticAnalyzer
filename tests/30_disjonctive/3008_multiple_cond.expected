tests/30_disjunctive/3008_multiple_cond.c:11.2-12: [ x in {[1;1] ∨ [4;4] ∨ [7;7]} ]
analysis ended